NitroxClient/GameLogic/Entities.cs

```diff
@@ -205,9 +205,9 @@ private void AddPendingParentEntity(Entity entity)
         // to move on to other spawning and come back when this item is ready.  
         private IEnumerator AwaitAnyRequiredEntitySetup(GameObject gameObject)
```

Comment:
This code is removed per: https://github.com/SubnauticaNitrox/Nitrox/pull/1999 Would like to get that merged.